packages/fast-usdc/src/types.ts

```diff
@@ -62,9 +62,13 @@ export interface PendingTx extends CctpTxEvidence {
 }
 
 export type FeeConfig = {
+  /** flat fee charged for every advance */
   flat: Amount<'nat'>;
+  /** percentage charged for every advance */
```

Comment:
thanks for these explanations. please be a bit more explicit.
```suggestion
  /** proportion of advance kept as a fee */
```